QSB/Animation/QSBNetworkAnimator.cs

```diff
@@ -0,0 +1,375 @@
+﻿using QSB.Animation.Events;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace QSB.Animation
+{
+    // Cleaned up unity code. UNET is so broken I gave up and fixed it myself.
```

Comment:
That's great! :)

For later, we could check out https://github.com/vis2k/Mirror, it's a community edition of UNet, cleaned up and allegedly with fewer bugs.